.travis.yml

```diff
@@ -68,7 +68,16 @@ install:
     fi
   # Always install from pypi
   - pip install $PRE pep8 cycler
+  # Remove this once we can run all tests in pytest
   - 'pip install https://github.com/tacaswell/nose/zipball/mnt_py36_compat#egg=nose'
+  - pip install pytest
+
+  # some handy plugins for pytest, is this the right place to install them?
+  # https://pypi.python.org/pypi/pytest-cov for docs
+  - pip install pytest-cov
+  # https://pypi.python.org/pypi/pytest-flake8 for docs
+  - pip install pytest-flake8
```

Comment:
flake8 finds a lot more problems than pep8 does as a tool.  And we should carefully consider whether we want to add the burden of checking for those things at this time.  That can always be a follow-on to this after some discussion.

Also, with respect to PEP8 checking, ensure that we're checking the same set of files (we don't check all of them) and ignoring the same set of violations.